contracts/test/foundry/FleekERC721/Mint.t.sol

```diff
@@ -35,13 +35,31 @@ contract Test_FleekERC721_Mint is Test_FleekERC721_Base {
             "94e8ba38568aea4fb277a37a4c472d94a6ce880a",
             "https://github.com/a-different/repository",
             TestConstants.LOGO_1,
-            0x654321
+            0x654321,
+            false
         );
 
         assertEq(firstMint, 0);
         assertEq(secondMint, 1);
     }
 
+    function test_mintWithAutoApprovalAPsOn() public {
+        uint256 mint = CuT.mint(
+            address(12),
+            "Different App Name",
+            "This is a different description for another app.",
+            "https://fleek.xyz",
+            "fleek.eth",
+            "94e8ba38568aea4fb277a37a4c472d94a6ce880a",
+            "https://github.com/a-different/repository",
+            TestConstants.LOGO_1,
+            0x654321,
+            true
+        );
+
+        assertEq(mint, 0);
```

Comment:
Should we also assert that auto approval is set to true?

Reply:
That's tested in AP test files.